_posts/fr/newsletters/2024-10-11-newsletter_fr.md

```diff
@@ -0,0 +1,271 @@
+---
+title: 'Bulletin Hebdomadaire Bitcoin Optech #324'
+permalink: /fr/newsletters/2024/10/11/
+name: 2024-10-11-newsletter-fr
+slug: 2024-10-11-newsletter-fr
+type: newsletter
+layout: newsletter
+lang: fr
+---
+Le bulletin de cette semaine annonce trois vulnérabilités affectant les anciennes versions du nœud
+complet Bitcoin Core, annonce une vulnérabilité distincte affectant les anciennes versions du nœud
+complet btcd, et inclut une contribution pour un guide Optech décrivant comment utiliser plusieurs nouvelles
+fonctionnalités du réseau P2P ajoutées dans Bitcoin Core 28.0. Sont également inclus nos sections
+habituelles avec le résumé d'une réunion du Bitcoin Core PR Review Club,
+l'annonce des mises à jour et des versions candidates, et présente les changements
+apportés aux principaux logiciels d'infrastructure Bitcoin.
+
+## Nouvelles
+
+- **Divulgation de vulnérabilités affectant les versions de Bitcoin Core antérieures à 25.0 :**
+  Niklas Gögge a [posté][gogge corevuln] sur la liste de diffusion Bitcoin-Dev
+  des liens vers les annonces de trois vulnérabilités affectant les versions
+  de Bitcoin Core qui sont obsolètes depuis au moins avril 2024.
+
+  - [CVE-2024-35202 vulnérabilité de crash à distance][] : un attaquant peut envoyer
+    un message de [bloc compact][topic compact block relay] conçu délibérément
+    pour faire échouer la reconstruction du bloc. Les échecs de reconstruction
+    se produisent parfois même dans l'utilisation honnête du protocole, dans ce cas
+    le nœud récepteur demande le bloc complet.
+
+    Cependant, au lieu de répondre avec un bloc complet, l'attaquant pourrait
+    envoyer un second message de bloc compact pour le même en-tête de bloc.
+    Avant Bitcoin Core 25.0, cela provoquerait le crash du nœud, car le
+    code était conçu pour empêcher le code de reconstruction de bloc compact
+    de s'exécuter plus d'une fois sur la même session de bloc compact.
+
+    Cette vulnérabilité facilement exploitable aurait pu être utilisée pour faire crasher
+    n'importe quel nœud Bitcoin Core, ce qui pourrait être utilisé comme partie d'autres
+    attaques pour voler de l'argent aux utilisateurs. Par exemple, un nœud Bitcoin Core crashé
+    serait incapable d'alerter un nœud LN connecté qu'une contrepartie de canal tentait de voler
```

Comment:
Je ne sais pas si c'est important mais l'original dit: "channel counterparty was attempting to steal funds"

Du coup "tentait de voler" peut aussi être "été tenté pour voler des fonds". 
Je ne suis pas sûr que le sens de ces deux phrases soit le même, c'est pour ça que je met ce commentaire ici.